mpsl/fem/CMakeLists.txt

```diff
@@ -4,7 +4,17 @@
 #
 
 function(mpsl_fem_append_lib fem_lib_name)
-    set(LIB_DIR "${CMAKE_CURRENT_SOURCE_DIR}/${fem_lib_name}/lib/${CONFIG_MPSL_LIB_DIR}/${CONFIG_MPSL_LIB_FLOAT_ABI_DIR}")
```

Comment:
@hermabe , I assume these changes are unintentional?

Reply:
I didnt think about this file being defined in internal mpsl repo and exported. The changes I did need to be backported.

Reply:
Dropped this change

Reply:
Created an internal backport PR